ios/fastlane/actions/project_name.rb

```diff
@@ -74,7 +74,7 @@ def self.default_project_filename
       def self.project_filename
         env_info = Actions::GetEnvironmentInfoAction.run({})
         if env_info[PROJECT_NAME_ENV_KEY]
-          env_info[PROJECT_NAME_ENV_KEY] + env_info[PROJECT_EXTENSION_ENV_KEY] || DEFAULT_PROJECT_EXTENSION
+          ENV['PWD'] + "/" + env_info[PROJECT_NAME_ENV_KEY] + (env_info[PROJECT_EXTENSION_ENV_KEY] || DEFAULT_PROJECT_EXTENSION)
```

Comment:
Did you know what is this?  It is like a temporal fix for a problem that we had with Bitrise, that when we have the same env var set in the shell env (Bitrise env in this case) than in the env files, shell envs are ignored. And this temporary fix is for overriding the env var from the env file with the one set in the shell env. 
Would be great to have a feature for overriding all envs by shell envs when they are equals. You can do it on the `get_environment_info` action

Reply:
@sfernandez11 any thoughts about this?